axelrod/strategies/axelrod_first.py

```diff
@@ -56,97 +69,203 @@ def strategy(self, opponent: Player) -> Action:
         opponent ever plays D."""
         if len(self.history) < self._rounds_to_cooperate:
             return C
-        if opponent.defections:
+        if opponent.defections > 0:  # Implement Grudger
             return D
         return C
 
+class FirstByDowning(Player):
+    """
+    Submitted to Axelrod's first tournament by Downing
+
+    The description written in [Axelrod1980]_ is:
+
+    > "This rule selects its choice to maximize its own longterm expected payoff on
+    > the assumption that the other rule cooperates with a fixed probability which
+    > depends only on whether the other player cooperated or defected on the previous
+    > move. These two probabilities estimates are continuously updated as the game
+    > progresses. Initially, they are both assumed to be .5, which amounts to the
+    > pessimistic assumption that the other player is not responsive. This rule is
+    > based on an outcome maximization interpretation of human performances proposed
+    > by Downing (1975)."
+
+    The Downing (1975) paper is "The Prisoner's Dilemma Game as a
+    Problem-Solving Phenomenon" [Downing1975]_ and this is used to implement the
+    strategy.
+
+    There are a number of specific points in this paper, on page 371:
+
+    > "[...] In these strategies, O's [the opponent's] response on trial N is in
+    some way dependent or contingent on S's [the subject's] response on trial N-
+    1. All varieties of these lag-one matching strategies can be defined by two
+    parameters: the conditional probability that O will choose C following C by
+    S, P(C_o | C_s) and the conditional probability that O will choose C
+    following D by S, P(C_o, D_s)."
+
+    Throughout the paper the strategy (S) assumes that the opponent (D) is
+    playing a reactive strategy defined by these two conditional probabilities.
+
+    The strategy aims to maximise the long run utility against such a strategy
+    and the mechanism for this is described in Appendix A (more on this later).
+
+    One final point from the main text is, on page 372:
+
+    > "For the various lag-one matching strategies of O, the maximizing
+    strategies of S will be 100% C, or 100% D, or for some strategies all S
+    strategies will be functionaly equivalent."
```

Comment:
possible typo: "functionaly" (If this is how it's actually spelt in the paper, we should add (sic) here)